docs-release-notes.md

```diff
@@ -1,3 +1,39 @@
+# February 23, 2024
+
+## Device Sync
+
+- Configure Sync/Sync Settings: Fix small typo.
+
+## Functions
+
+- Landing Page: Fix typo.
+- Test Function: Fix typo.
+
+## Triggers
+
+- Send Trigger Events to AWS EventBridge: Update page with new sections on error handling for Database Triggers.
+
+## Monitor App Activity
+
+- View Application Logs: Update page with new `trigger_error_handler` CLI log type in support of AWS EventBridge custom error handling.
+- Forward Logs to a Service: Update list of supported CLI log types with new `trigger_error_handler` type in support of AWS EventBridge custom error handling.
+- Log Types/Trigger Error Handler Logs: New page documenting the trigger error handler log type in support of AWS EventBridge custom error handling.
+
+## Reference
+
+- App Configuration Files/Triggers: Update database trigger configuration, most importantly adding `event_processors` and `error_handler` configuration.
+- App Services Admin API
```

Comment:
We're missing a release note for this PR from Nick that added two new endpoints, and also updated several of the existing schema definitions: https://github.com/mongodb/docs-app-services/pull/625